addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13683,7 +13683,10 @@ msgctxt "#24047"
 msgid "This add-on can't be uninstalled"
 msgstr ""
 
-#empty string with id 24048
+#: xbmc/addons/GUIDialogAddonInfo.cpp
+msgctxt "#24048"
+msgid "%s is a system add-on"
```

Comment:
it's in line with the message shown when you try disabling/uninstalling a dependency. on an additional line you get "this addon can't be uninstalled" or "this addon can't be disabled" depends on what you try doing.